frontend/src/components/ViewTrips/delete-trip-button.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import app from '../Firebase/';
+import Button from 'react-bootstrap/Button';
+
+import * as DB from '../../constants/database.js';
+
+const db = app.firestore();
+const LIMIT_QUERY_DOCS_RETRIEVED = 5;
+
+/**
+ * Component used to delete a Trip.
+ *
+ *
+ * @param {Object} props These are the props for this component:
+ * - tripId: Document ID for the current Trip document.
+ * - refreshTripsContainer: Handler that refreshes the TripsContainer
+ *        component upon trip creation (Remove when fix Issue #62).
+ */
+const DeleteTripsButton = (props) => {
+   /**
+   * Deletes documents in query with a batch delete.
+   *
+   * This was taken from the delete collection snippets in the documentation
+   * at https://firebase.google.com/docs/firestore/manage-data/delete-data.
+    *
+    * @param {firebase.firestore.Firestore} db Firestore database instance.
+    * @param {firebase.firestore.Query} query Query containing documents from
+    *     the activities subcollection of a trip documents.
+    * @param {Function} resolve Resolve function that returns a void Promise.
+    */
```

Comment:
Align indentation across all lines of this block comment.

Reply:
Oh man that was gross. Fixed.